### ISSUE TITLE
Does not work in Laravel 10

### ISSUE THREAD

**Issue by israelcena** [ryoluo/sail-ssl#7]
When I've tryed install in my Laravel 10 app, my shell console returns:

```
$ sail composer require ryoluo/sail-ssl --dev
Info from https://repo.packagist.org: #StandWithUkraine
./composer.json has been updated
Running composer update ryoluo/sail-ssl
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires ryoluo/sail-ssl * -> satisfiable by ryoluo/sail-ssl[v1.0.0, ..., v1.1.1].
    - ryoluo/sail-ssl[v1.0.0, ..., v1.1.1] require illuminate/console ^8.0|^9.0 -> found illuminate/console[v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.0] but these were not loaded, likely because it conflicts with another require.

You can also try re-running composer require with an explicit version constraint, e.g. "composer require ryoluo/sail-ssl:*" to figure out if any version is installable, or "composer require ryoluo/sail-ssl:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

**Comment by janar153** [comment]
I have made Pull Request to add support for Laravel 10 - https://github.com/ryoluo/sail-ssl/pull/8

**Comment by ryoluo** [comment]
I've just released [v1.2.0](https://github.com/ryoluo/sail-ssl/releases/tag/v1.2.0) that is compatible with Laravel 10.
Thank you very much @israelcena san and @janar153 san

**Comment by israelcena** [comment]
Thank you for release v1.2.0 @ryoluo
